<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCommandDialog</class>
 <widget class="QDialog" name="EditCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit command</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border: 1px solid #222;
}</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QComboBox" name="commandTypeBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Map Command</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Console Command</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="nameText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameText</tabstop>
  <tabstop>commandTypeBox</tabstop>
  <tabstop>commandText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditCommandDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
